splot/_viz_utils.py

```diff
@@ -245,7 +245,7 @@ def shift_colormap(cmap, start=0, midpoint=0.5, stop=1.0, name='shiftedcmap'):
 
     Parameters
     ----------
-    cmap : matplotlib colormap
+    cmap : str or matplotlib.cm instance
```

Comment:
Just a preference note: when labeling types I think it looks good to put the types in `set` brackets. For example:
```python
some_parameter : {int, float}
    This is some parameter. Default is None.
```
rather than
```python
some_parameter : int or float
    This is some parameter. Default is None.
```

Again, this is just my preference/suggestion, but you (and others) may completely disagree.